packages/cli/src/lib/graphql/admin/list-storefronts.ts

```diff
@@ -26,6 +29,23 @@ interface HydrogenStorefront {
   currentProductionDeployment: Deployment | null;
 }
 
-export interface ListStorefrontsSchema {
+interface ListStorefrontsSchema {
   hydrogenStorefronts: HydrogenStorefront[];
 }
+
+export async function getStorefrontsWithDeployment(shop: string) {
```

Comment:
`getStorefrontsWithDeployment` is an interesting name for this! Why `WithDeployment`?

Reply:
Because I think that's the only difference between this function and the one from `link-...`. This one adds `currentProductionDeployment` to the query.

I guess this could also be `getStorefrontsForListing` vs `getStorefrontsForLinking`?